superTuxKart: 1.4 -> 1.5

SchweGELBin opened the pull request:
<!--
^ Please summarise the changes you have done and explain why they are necessary here ^

For package updates please link to a changelog or describe changes, this helps your fellow maintainers discover breaking updates.
For new packages please briefly describe the package or provide a link to its homepage.
-->

https://github.com/supertuxkart/stk-code/releases/tag/1.5
I've also updated the assets to the newest version.

## Things done

<!-- Please check what applies. Note that these are not hard requirements but merely serve as information for reviewers. -->

- Built on platform:
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- Tested, as applicable:
  - [ ] [NixOS tests] in [nixos/tests].
  - [ ] [Package tests] at `passthru.tests`.
  - [ ] Tests in [lib/tests] or [pkgs/test] for functions and "core" functionality.
- [ ] Ran `nixpkgs-review` on this PR. See [nixpkgs-review usage].
- [x] Tested basic functionality of all binary files, usually in `./result/bin/`.
- Nixpkgs Release Notes
  - [ ] Package update: when the change is major or breaking.
- NixOS Release Notes
  - [ ] Module addition: when adding a new NixOS module.
  - [ ] Module update: when the change is significant.
- [x] Fits [CONTRIBUTING.md], [pkgs/README.md], [maintainers/README.md] and other READMEs.

[NixOS tests]: https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests
[Package tests]: https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-tests
[nixpkgs-review usage]: https://github.com/Mic92/nixpkgs-review#usage

[CONTRIBUTING.md]: https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md
[lib/tests]: https://github.com/NixOS/nixpkgs/blob/master/lib/tests
[maintainers/README.md]: https://github.com/NixOS/nixpkgs/blob/master/maintainers/README.md
[nixos/tests]: https://github.com/NixOS/nixpkgs/blob/master/nixos/tests
[pkgs/README.md]: https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md
[pkgs/test]: https://github.com/NixOS/nixpkgs/blob/master/pkgs/test

---

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

SchweGELBin committed:
superTuxKart: 1.4 -> 1.5

stk-assets: 18464 -> 18621

SchweGELBin committed:
superTuxKart: update descriptions

The decription was copied from the official website's discover page ([here](https://supertuxkart.net/Discover)) and got modified.
It got added by @Benau [here](https://github.com/supertuxkart/stk-website/commit/2c46fee5aad8412a0187d4e2e0580a5403fbdbf6)

SchweGELBin committed:
superTuxKart: add SchweGELBin to maintainers

SchweGELBin committed:
superTuxKart: rename src.rev to src.tag

marcin-serwin commented on pkgs/by-name/su/superTuxKart/package.nix:
```diff
@@ -162,17 +147,19 @@ stdenv.mkDerivation rec {
+      and provide an enjoyable experience for all ages.
     '';
     homepage = "https://supertuxkart.net/";
     license = lib.licenses.gpl2Plus;
```
It looks like the license has been updated at some point.
```suggestion
    license = lib.licenses.gpl3Plus;
```

SchweGELBin replied:
Oh, you're right.

marcin-serwin reviewed (approved):
Builds and runs on `aarch64-linux`

SchweGELBin committed:
superTuxKart: update license from gpl2Plus to gpl3Plus

The source code is licensed under GPLv3 or later.
This was changed upstream at Nov 23, 2018 [here](https://github.com/supertuxkart/stk-code/commit/06952bfd524ce368e12e62aadcc8546b0786051c),
but wasn't updated here in nixpkgs, until now.

The game assets are released under a mixture of licenses and noted in their respective 'licenses.txt' files.

Yarny0 reviewed (approved):
#### Review per https://github.com/NixOS/nixpkgs/tree/6a08e6bb4e46ff7fcbb53d409b253f6bad8a28ce/pkgs#package-updates

##### Reviewed points

Done after rebasing the branch onto [current `nixos-unstable`](https://github.com/NixOS/nixpkgs/tree/6a08e6bb4e46ff7fcbb53d409b253f6bad8a28ce):

- [x] package name fits guidelines
- [x] package version fits guidelines
- [x] package builds on *x86_64-linux*
- [x] executables tested on *x86_64-linux*
- [x] any change of upstream are verified (*none*)
- [x] the motives for any special packaging choices are documented (*none*)
- [x] all depending packages build  (*see below*)
- [x] patches have a comment describing either the upstream URL or a reason why the patch wasn't upstreamed (*none*)
- [x] patches that are remotely available are fetched rather than vendored (*none*)
- [x] I raced through "Hacienca" ... and won!

##### Possible improvements

- The author names in the commits are apparently not consistent (~~"Miche"~~ "Michi" vs. "SchweGELBin").  Is that intended?
- `gpl3Plus` license also applies to the "old" version 1.4.  So it would be more correct to move that commit to the base of the branch.  Then it is documented that the license change is a fix in nixpkgs and not a consequence of the version update.
- The `longDescription` (in particular "Our aim...") reads like it was copied from another web page.  I don't expect that being a problem (but I am not a lawyer).  It would be good to cite the source in the commit message.  And we might rephrase it so it is written more from the nixpkgs perspective.

But I don't see any of those poits as an obstacle to get the pull request merged.


#### `nixpkgs-review` result

Generated using [`nixpkgs-review`](https://github.com/Mic92/nixpkgs-review).

Command: `nixpkgs-review`
Commit: `e9fe48e2fec353986b4a6ccfe92f70b3fad84f7f`

---
##### `x86_64-linux`
<details>
  <summary>:white_check_mark: 1 package built:</summary>
  <ul>
    <li>superTuxKart</li>
  </ul>
</details>

---

Edit: Fixed typo in hint about commit author names.

Yarny0 commented on pkgs/by-name/su/superTuxKart/package.nix:
```diff
@@ -29,8 +28,8 @@
-    rev = "18464";
-    sha256 = "1a84j3psl4cxzkn5ynakpjill7i2f9ki2p729bpmbrvg8fki95aa";
+    rev = "18621";
+    sha256 = "sha256-iqQSezGu0tecA53qhrtYA77SLj28WUUCcL4ZCJbK5C8=";
```
Then we should use this opportunity to switch to `hash` which seems perferred in nixpkgs.

(But again, I consider this a non-blocking suggestion for the pull request.)
